test_scripts/Defects/5_0/2448_cleansing_app_info_after_MASTER_RESET_FACTORY_DEFAULTS/2448_4_FACTORY_DEFAULTS_no_app_during_shutting_down.lua

```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2448
+--
+-- Steps:
+-- 1. HMI and SDL are started
+-- 2. Mobile app is registered
+-- 3. App is in FULL HMI level
+-- 4. PTU is performed successfully
+-- 5. AddSubMenu is added
+-- 6. App is disconnected
+-- 7. HMI sends BC.OnExitAllApplications(reason = FACTORY_DEFAULTS)
+-- SDL does:
+--  a. clear app's persist data
+--  b. stop working
+-- 8. HMI and SDL are started
+-- 9. App registers with actual hashID
+-- SDL does:
+--  a. register app with resultCode RESUME_FAILED
+--  b. not resume HMI level
```

Comment:
If the difference between MASTER_RESET and FACTORY_DEFAULTS is persist of policyDB this need to be mentioned in description. E.g. PTU is started since PT was re-initialized or PTU shouldn't be started since PT was retained.